apps/site/vercel.json

```
@@ -1,4 +1,4 @@
 {
   "buildCommand": "turbo build",
-  "installCommand": "npm install --omit=dev --include-workspace-root"
+  "installCommand": "pnpm install --prod"
```

```suggestion
  "installCommand": "pnpm install --prod --forzen-lockfile"
```
Being explicit like we are elsewhere